compiler/test/dotty/tools/dotc/reporting/ErrorMessagesTests.scala

```diff
@@ -963,13 +963,9 @@ class ErrorMessagesTests extends ErrorMessagesTest {
         |}
       """.stripMargin
     }.expect { (ictx, messages) =>
-      implicit val ctx: Context = ictx
```

Comment:
`defn` was not used but I think you still need the implicit `ctx`. It is passed implicitly to the method `show`